recipes/python-sa-gwdata/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "python-sa-gwdata" %}
+{% set version = "0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2a2ae3b98f2fd125f34b270d053a39383ee39d3820e0335bf4a93aeff88901ca
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools_scm
+  run:
+    - python >=3.7
+    - requests
+    - platformdirs
+    - pyarrow
+    - pyshp
+    - shapely
+    - contextily
+    - geopandas
```

The package metadata has different dependencies here. Please check upstream.

```suggestion
    - python >=3.7
    - requests
    - pandas >=0.24.1
    - platformdirs
    - pyarrow
    - pyshp
```